app/views/idv/questions/index.html.slim

```diff
@@ -0,0 +1,4 @@
+- if @question.choices.nil?
+  = render 'idv/questions/text'
+- else
+  = render 'idv/questions/multiple_choice'
```

Comment:
Since this is pure logic would it make more sense in the controller? e.g:

```
def index
  if @question.choices.nil?
    render :question_multi_choice
  else
    render :question_text
```